### PR TITLE
reorg of scripts and semi-split of codalib/codadaemon

### PR DESCRIPTION
**Pull request by jkrauska**
first pass at splitting up some of scripts

### CODE REVIEW

File: scripts/codadaemon.py [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+                   threads=8)
+
+# --------------------------------------------------------------------------------
+
+
+""" functional programming!  -- loop that runs a function once per block """
+def each_block(fnc, maxblock=999, pretty=False):
+    last_block = None
+    while(True):
+        current_block = coda_block()
```
Comment (yourbuddyconner) [inline scripts/codadaemon.py]: I think you need to import codalib still as `coda_block` and `coda_slot` will be undefined.

File: scripts/transactions.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 else:
     big_key = '%s/testkeys/high0' % (os.getcwd())
 
-clientport = codalib.coda_processes()[0]
+clientport = 8301
```
Comment (yourbuddyconner) [inline scripts/transactions.py]: Might break stuff down the line to have this hardcoded but ¯\_(ツ)_/¯

File: scripts/codadaemon.py (hunk 1 above)
Comment (jkrauska) [reply scripts/codadaemon.py]: yes!

File: scripts/transactions.py (hunk 2 above)
Comment (jkrauska) [reply scripts/transactions.py]: referencing lib default instead.

**Pull request merged**